File: snippets/fsharp/System/TupleT1,T2,T3,T4,T5,T6,T7,TRest/Overview/octuple1.fs
﻿open System

// <Snippet1>
let primes = new Tuple<Int32, Int32, Int32, Int32, Int32, Int32, Int32,  
               Tuple<Int32>> (2, 3, 5, 7, 11, 13, 17, new Tuple<Int32>(19))
// </Snippet1>
printfn $"{primes}"

File: snippets/csharp/System/TupleT1,T2,T3,T4,T5,T6,T7,TRest/Overview/octuple1.cs
using System;

public class Class1
{
    public static void Main()
    {
        // <Snippet1>
        var primes = new Tuple<int, int, int, int, int, int, int,
                     Tuple<int>>(2, 3, 5, 7, 11, 13, 17, new Tuple<int>(19));
        // </Snippet1>
        Console.WriteLine(primes);
    }
}
